src/ontology/fbbt-edit.obo

```diff
@@ -118131,6 +118131,145 @@ relationship: develops_from FBbt:00004945 ! onion stage spermatid
 created_by: camcur
 creation_date: 2008-05-15T05:24:58Z
 
+[Term]
+id: FBbt:00058000
+name: adult class I enteroendocrine cell
+def: "Enteroendocrine cell of the adult midgut expressing the hormone Allatostatin C. These cells are found throughout the midgut and co-express different other hormones depending on their exact localization (Chen et al., 2016; Guo et al., 2018)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00047149 ! adult enteroendocrine cell
+intersection_of: expresses http://flybase.org/reports/FBgn0032336 ! AstC
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:28:47Z
+
+[Term]
+id: FBbt:00058001
+name: adult anterior midgut class I enteroendocrine cell
+def: "Class I enteroendocrine cell localized in the anterior midgut of the adult. These cells express Orcokinin B in addition to Allatostatin C (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00058000 ! adult class I enteroendocrine cell
+intersection_of: part_of FBbt:00047154 ! adult anterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0034935 ! Orcokinin
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:42:19Z
+
+[Term]
+id: FBbt:00058002
+name: adult middle midgut class I enteroendocrine cell
+def: "Class I enteroendocrine cell localized in the middle midgut of the adult. These cells express Orcokinin B in addition to Allatostatin C (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00058000 ! adult class I enteroendocrine cell
+intersection_of: part_of FBbt:00110897 ! adult midgut region R3
+relationship: expresses http://flybase.org/reports/FBgn0034935 ! Orcokinin
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:44:32Z
+
+[Term]
+id: FBbt:00058003
+name: adult posterior midgut class I enteroendocrine cell
+def: "Class I enteroendocrine cell localized in the posterior midgut of the adult. These cells express Allatostatin A and Allatostatin B/Mip in addition to Allatostatin C (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
```

Comment:
Minor issue, but I would say 'Allatostatin B (Mip)' to be clearer that this is a synonym and you don't mean 'Allatostatin B and/or Mip'.